packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeDetailsItem/MT5TradeDetailsItem.tsx

```diff
@@ -25,15 +25,17 @@ const MT5TradeDetailsItem: FC<TMT5TradeDetailsItemProps> = ({ className, label,
             <Text colorStyle='subtle' size='sm'>
                 {label}
             </Text>
-            <div className='flex items-center space-x-400 pr-400'>
+            <div className='flex items-center gap-x-400'>
                 <Text bold size='sm'>
                     {value}
                 </Text>
                 {variant === 'clipboard' && <Clipboard textCopy={value} />}
                 {variant === 'password' && (
                     <Tooltip alignment='left' isVisible={isHovered && isDesktop} message='Change password'>
                         <div ref={hoverRef}>
-                            <EditIcon className='cursor-pointer' />
+                            <Button colorStyle='prominent' size='sm'>
```

Comment:
Same here, color does not exist

Reply:
@sergei-deriv you can refer to the storybook that I send over in our channel regarding the props. Or you can view it [here](https://quill-design-6v3.pages.dev/?path=/docs/buttons-basic--docs)